joint_state_publisher/package.xml

```diff
@@ -18,12 +18,6 @@
   <exec_depend>sensor_msgs</exec_depend>
   <exec_depend>std_msgs</exec_depend>
 
-  <test_depend>ament_copyright</test_depend>
-  <test_depend>ament_xmllint</test_depend>
```

Comment:
I think `ament_copyright` and `ament_xmllint` tests still work. I would recommend keeping them. It's just the launch tests that are broken.

Reply:
Done in b882325

Reply:
Almost done :wink: The `test_copyright.py` and `test_xmllint.py` files need to be un-deleted. Maybe the `tests_require=['pytest'],` line in `setup.py` is needed too, but I'm not 100% sure.

Reply:
Bah, of course.  Sorry, went too fast, and we obviously need that.  Fixed and force-pushed now, thanks.